### ISSUE TITLE
loadModel() function currently wont accept a name argument

### ISSUE THREAD

**Issue by kevinkmcguigan** [NSCC-COGS/Aestheta#82]
https://github.com/NSCC-COGS/Aestheta/blob/229dbbfe474a2a7740f5e8bf74b0633c80c5feec/Library/core.py#L108

The function works when no name is set by finding the most recent available model in the /Models directory in the git repo:
https://github.com/NSCC-COGS/Aestheta/tree/main/Models

However, if an actual model name is specified - the loadModel() function fails. 

**Add the basic functionality to specify a model by name in the loadModel() function.** 
_Optionally: test to see if the specified named model exists  - or add other features like finding the SECOND most recent model etc..._

**Comment by kevinkmcguigan** [comment]
Hey @donuty-party, thanks for addressing this. My intention was to see a simple solution where we could load by the explicit model name for now and then build functionality from there as required. I see you've set up a system so we can be flexible with it in the future. This may work for us! Perhaps we will need to begin brainstorming some of the various attributes that will change across our modes. It seems you've _gone to town!_

Just a rough thought on variables and attributes we way want to consider capturing in the model class - models may vary by:

-  model type (eg, scikit gradient boost vs XGboost vs keras neural network) 
-  model parameters (which are dependent on model type)
-  input data type (ie what data/array structure the model will support)
-  what data the model as trained on
- what classification standard the model outputs 

Perhaps our model class/objects could contain some instructions and validation information as well.
Just a note to be aware of -some functionality in terms of capturing model information exists in, for example, [the model class in keras](https://keras.io/api/models/model/#model-class) - such as the model summary. We may need to learn some of this as we go. 

We can discuss in more detail on Tuesday.

**Comment by donuty-party** [comment]
Reverted my changes per #102 and moved them to the 82-load-model-by-name branch.
